### ISSUE TITLE
Invalid database version in v2.0.0-rc2 

### ISSUE THREAD

**Issue by ambyte** [mbdavid/LiteDB#233]
In v2.0.0-rc2 you have changed db version and now I get an error
LiteDB.LiteException: Invalid database version: 5
Before that I used v2.0.0-rc

What I must do for fix it?

**Comment by mbdavid** [comment]
Hi @ambyte, datafile change in this last RC2 version (from RC and beta). So, I update internal file version check.
To upgrade, use LiteDB.Shell (from RC) to export data and with LiteDB.Shell (from RC2) use to import in another datafile.

In shell RC/Beta, use:
```
open beta-rc.db
dump > mydump.dmp
```

In shell RC2
```
open new-datafile.db
dump < mydump.dmp
```

I will add this in wiki documentation.

**Comment by negue** [comment]
Haven't ran into this kind of issue, but what if the database version changes from LiteDB2.0 to 2.X , and I upgrade my app to it, how to migrate then?

I don't know if its possible but maybe we need an self migrating option without using LiteDB.Shell, for example, on Android its impossible to run the LiteDB.Shell.exe

**Comment by mbdavid** [comment]
@negue, this will not happen (or, at lease, if anyone found a bug that needs change). 

From rc to final release, an issue posted about DateTime field indexed, I found a "little" bug that I fixed. In this case, datafile must be updated. In a 2.X scenario, will force me to write a "fix datafile". It´s a very hard job, but I will need do if any problem in data structure are found.

To next major version (v3) datafile will be changed again (to support anothers types of indexes). An portable update will be made.
